### PR TITLE
Add configuration, npm packages and layout component

### PR DESCRIPTION
**Pull request by mathiasbno**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  "devDependencies": {
+    "babel-core": "^5.8.23",
+    "babel-eslint": "^4.1.1",
+    "babel-loader": "^5.3.2",
+    "css-loader": "^0.17.0",
+    "eslint": "^1.5.1",
+    "eslint-config-airbnb": "^0.1.0",
+    "eslint-config-hyperoslo": "^1.0.0",
+    "eslint-loader": "^1.0.0",
+    "eslint-plugin-react": "^3.4.2",
```
Comment (sindrenm) [inline package.json]: This is a pretty old version, isn't it? The `react/sort-comp` rule that you've disabled in your ESLint config has also been fixed in https://github.com/yannickcr/eslint-plugin-react/pull/429, which should be present in the latest version. Any reason you're not updating?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    "lint": "eslint src/**/*.{js,jsx} *.js; exit 0",
+    "release": "webpack --progress --optimize-minimize",
+    "serve": "webpack-dev-server --port $PORT",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hyperoslo/web-team-tv.git"
+  },
```
Comment (sindrenm) [inline package.json]: This entire config can be [simplified](https://docs.npmjs.com/files/package.json#repository):

``` json
"repository": "hyperoslo/web-team-tv"
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "web-team-tv",
+  "version": "1.0.0",
```
Comment (sindrenm) [inline package.json]: That was fast!

File: package.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    "eslint": "^1.5.1",
+    "eslint-config-airbnb": "^0.1.0",
+    "eslint-config-hyperoslo": "^1.0.0",
+    "eslint-loader": "^1.0.0",
+    "eslint-plugin-react": "^3.4.2",
+    "file-loader": "^0.8.4",
+    "html-webpack-plugin": "^1.6.1",
+    "normalize.css": "^3.0.3",
+    "style-loader": "^0.12.3",
+    "stylus": "^0.52.4",
```
Comment (sindrenm) [inline package.json]: :heart:

File: webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    filename: 'bundle.js',
+  },
+  resolve: {
+    extensions: ['', '.js', '.jsx'],
+  },
+  module: {
+    loaders: [
+      {
+        test: /\.(png|jpg|otf)$/,
+        loader: 'url-loader?limit=100000',
```
Comment (sindrenm) [inline webpack.config.js:17]: image

File: package.json (hunk 1 above)
Comment (mathiasbno) [inline package.json]: Yes, it will break all my things :( And it is, upgrading is the worst.

**Comment by sindrenm** [comment]
Awesome!

**Pull request merged**